troubleshooting/known_issues/ims_images_orphaned_in_s3.md

```diff
@@ -0,0 +1,242 @@
+# Known Issue: IMS Images Orphaned in S3
+
+There was an issue where image files larger than 5Gb were not correctly deleted from S3 when the image
+was deleted through the IMS service. These artifacts are no longer referenced by IMS but are still
+left in the S3 `boot-images` bucket and need to be deleted manually.
+
+## Fix
+
+These items can be deleted out of S3. There is a script defined here that will automatically look through
+all of the artifacts in the `boot-images` bucket in S3 and finds the artifacts that no longer belong to
+a valid IMS image. There is a `--dry-run` option which will only list the artifacts found or without that
+option it will delete the orphaned artifacts.
+
+1. (`ncn-mw#`) Copy the script to a management node
+
+  Create a file named `orphaned_artifacts.py` on a management node with the following text in it.
+
+  ```python
```

Comment:
Recommend checking this script in directly, rather than having them copy/paste it.

Reply:
Fixed.